regtests/README.md

```diff
@@ -40,7 +40,7 @@ follows:
 
 ```shell
 ./gradlew clean :polaris-quarkus-server:assemble -Dquarkus.container-image.build=true --no-build-cache
-docker compose -f ./regtests/docker-compose.yml up --build --exit-code-from regtest
+docker-compose -f ./regtests/docker-compose.yml up --build --exit-code-from regtest
```

Comment:
This seems to be unrelated to the purpose of this PR. Please make this change in a separate PR.